src/components/accueil/PelicoProfilNavigation.tsx

```diff
@@ -0,0 +1,93 @@
+import { useRouter } from 'next/router';
+import React from 'react';
+
+import { Button } from '@mui/material';
+
+import { AvatarImg } from '../Avatar';
+import { CommentIcon } from '../activities/ActivityCard/CommentIcon';
+import { icons, DESC } from 'src/components/activities/utils';
+import { useActivities } from 'src/services/useActivities';
+import { primaryColor } from 'src/styles/variables.const';
+import { getUserDisplayName, toDate } from 'src/utils';
+import { ActivityType } from 'types/activity.type';
+import type { User } from 'types/user.type';
+
+export const PelicoProfilNavigation = ({ activeUser, displayAsUser = false }: { activeUser: User; displayAsUser?: boolean }) => {
```

Comment:
You can drop the displayAsUser here : 
`export const PelicoProfilNavigation = ({ activeUser }: { activeUser: User }) => {`